packages/@sanity/portable-text-editor/test/setup/collaborative.jest.env.ts

```diff
@@ -180,6 +185,7 @@ export default class CollaborationEnvironment extends NodeEnvironment {
               ])
             },
             pressKey: async (keyName: KeyInput, times?: number) => {
+              await delay(generateRandomInteger(0, 100))
```

Comment:
Can this potentially give random test failures that are hard to debug? Would it be better to use timer mocks here?

Reply:
This is to make it a bit more unpredictable actually, to test that the collaborative tests still pass even if there are random delays in the tests users keypresses.